src/hooks/offramp/useSubmitOfframp.ts

```diff
@@ -124,10 +126,10 @@ export const useSubmitOfframp = () => {
             setOfframpInitiating(false);
           }
         } catch (error) {
-          console.error('Error initializing the offramping process', error);
+          console.error('Error initializing the offramping process', (error as Error).message);
           // Display error message, differentiating between user rejection and other errors
-          if ((error as Error).message.includes('User rejected the request')) {
-            setInitializeFailed('Please switch to the correct network and try again.');
+          if ((error as Error).message.includes('User rejected')) {
+            showToast(ToastMessage.ERROR, 'You must sign the login request to offramp Argentine Peso');
```

Comment:
Nit: I guess we should stick to the 'Sell crypto' lingo and not mention offramps again. 
```suggestion
            showToast(ToastMessage.ERROR, 'You must sign the login request to be able to sell Argentine Peso');
```